### ISSUE TITLE
Execute_notebook assumes test location, incompatible with Gradescope

### ISSUE THREAD

**Issue by TylerADavis** [ucbds-infra/otter-grader#30]
`execute_notebook` [contains a line](https://github.com/ucbds-infra/otter-grader/blob/12066aee3293582aab459afa0d6c6eefdcd2e2c0/otter/grade.py#L190) that assumes all tests are in `/home/tests/`, regardless of how the notebook was originally configured. This is incompatible with the Gradescope autograder as Gradescope places all tests in `/autograder/source/tests/` and they are then copied to `/autograder/submission/tests` by the generated `run_autograder` script.

While the autograder is still able to run as a result of passing the [`tests_glob`](https://github.com/ucbds-infra/otter-grader/blob/12066aee3293582aab459afa0d6c6eefdcd2e2c0/otter/grade.py#L63) and `ignore_errors=True` (which results in the missing tests not printing errors) arguments, this results in all tests being run at the end of the notebook's execution, in the global namespace.

This would make the following sequence fail as all tests executed will see the value of x as 7.
```python
x = 5
grader.check("q1a") # check x = 5
x = x + 2
grader.check("q1b") # check x = 7
```

**Comment by chrispyles** [comment]
#31 fixes this issue; will merge PR and re-release on PyPI upon CI completion

**Comment by chrispyles** [comment]
Released v0.3.9
